### DISCUSSION TITLE
Add Core WCF to a existing MVC application.

### DISCUSSION THREAD

**Discussion by AlexCorsega** [CoreWCF/CoreWCF#1368] [General]
Is it possible that I can add Core WCF to an existing MVC application?
It's like adding the required middlewares and services of CoreWCF to the Program.cs of the MVC application.

My goal is to have the Core WCF project share the same domain as my MVC application. I know that it is possible by configuring the IIS server, but we must avoid that and instead do this at the application layer of our MVC application.

**Comment by g7ed6e** [comment]
Closing as duplicate of #1370
